### PR TITLE
Adding a couple of flags and line ending info

### PR DESCRIPTION
**Pull request by samsp99**
These were gotcha's to being able to use the tools and being able to develop a binding.

### CODE REVIEW

File: developers/index.md [hunk 1]
```diff
@@ -94,6 +94,7 @@ This script is specific for binding addons. Follow these steps to generate your
     ```bash
     create_openhab_binding_skeleton.cmd MyNewBinding "<Author>" <GitHubUsername>
     ```
+    Add your binding to the `bundles/pom.xml` file
```
Comment (Confectrician) [inline developers/index.md:97]: Please add a blank line between codeblock and text

File: developers/index.md [hunk 2]
```diff
@@ -124,3 +127,12 @@ This script is specific for binding addons. Follow these steps to generate your
...
 In case the new binding has dependencies to other bundles/libraries see [Adding Dependencies](buildsystem.html#adding-dependencies) for more information.
+
+If developing on Windows, git will convert line endings from LF to CRLF automatically. The repo uses LF,
+and spotless will object, and throw errors. If you use `mnv spotless:apply` to fix formatting, it will add CRLF
+to all files dirtying the entire branch. Adding a `.gitattributes` file to the openhab-addons folder with the
+following will tell git and spotless to use LF endings:
+
+```text
+* text eol=lf
+```
```
Comment (Confectrician) [inline developers/index.md:138]: Please also add a new line at the end.

**Review by Confectrician — COMMENTED** [review]
Hi @samsp99 and Thanks 👍

Two small remarks below.

File: developers/index.md (hunk 2 above)
Comment (samsp99) [reply developers/index.md:139]: There is, but its not showing in the diff UI 🤷‍♂️

**Comment by samsp99** [comment]
@Confectrician - should be good to go

**Review by Confectrician — APPROVED** [review]
Thanks 👍

**Comment by samsp99** [comment]
Do I need to do anything more?

**Comment by Confectrician** [comment]
No.

We have currently some problems with our homepage build and i would like to solve that first before merging this one.
When the build is working again it's the first one to merge. 🙂

**Pull request merged**